index.html

```diff
@@ -16,13 +17,16 @@
       }
       canvas {
         border: 1px solid #000;
+        display: inline-block;
```

Comment:
Everything inside <style> tag can be moved to index.css

Reply:
done